jdaviz/configs/default/plugins/model_fitting/initializers.py

```diff
@@ -190,7 +190,7 @@ def initialize(self, instance, x, y):
 
         # width can be estimated by the weighted
         # 2nd moment of the X coordinate.
-        dx = x - np.mean(x)
+        dx = x - np.nanmean(x)
```

Comment:
But `np.nanmean` would still return NaN if `x` is all NaNs. Is this something we need to handle?

Reply:
Same question for `y` also.

Reply:
But didn't you say multiprocessing would swallow the exception, causing confusion?

Reply:
I could imagine someone did a bad subset that encompass all NaNs in the cube and then try to fit it. It is *possible*?